modules/reactor/test/tests/meshgenerators/control_drum_mesh_generator/tests

```diff
@@ -0,0 +1,61 @@
+[Tests]
+  design = 'meshgenerators/ControlDrumMeshGenerator.md'
+  issues = '#28457'
+  [hex_with_pad]
```

Comment:
You can have a "regression test" sublock for all these tests and use the `requirement` and `detail` feature. Also, as mentioned in another comment, we need some `RunException` tests here to cover the error messages.

Reply:
Done